client/src/components/MapMain/MapMain.js

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import React from "react";
+import PropTypes from "prop-types";
+
+const Main = styled.main`
+  height: 60%;
+  flex: 1;
+  overflow: hidden;
+  position: relative;
+`;
+
+const MapMain = ({ children }) => {
+  return <Main>{children}</Main>;
+};
```

Comment:
You don't need an additional component. The styled component is enough :).
See https://github.com/mschmauder-hub/seriesmaster/pull/22#discussion_r488487739

Reply:
Buah, top! Did not know this! Thanks!